saas/axops/src/ui/src/app/views/+timeline/timeline/timeline.component.ts

```diff
@@ -195,10 +195,14 @@ export class TimelineComponent implements HasLayoutSettings, LayoutSettings, OnI
 
     private getFiltersByView(view) {
         this.currentView = view;
-        let filters = [];
+        if (this.toolbarFilters.data) {
```

Comment:
Is this if required?

Isn't this enough?
`this.toolbarFilters.data.length = 0;`

Reply:
this.toolbarFilters.data was initialized as 'undefined' at the beginning, but since it's changed to [] so I will change according to your comment.